tests/fixtures/test_fixtures.py

```diff
@@ -189,6 +192,44 @@ def test_load_users(app, db, admin_role):
     assert current_datastore.find_user(email="user@example.com")
 
 
+def test_load_communities(app, db):
+    dir_ = Path(__file__).parent
+    service = current_communities.service
+    communities = CommunitiesFixture(
+        [dir_ / "app_data", dir_.parent.parent / "invenio_rdm_records/fixtures/data"],
+        "communities.yaml",
+        dir_ / "app_data" / "img",
+    )
+
+    # mandatory for files bucket creation
+    location = Location(name="local", uri=str(dir_), default=True)
+    db.session.add(location)
+    db.session.commit()
```

Comment:
```suggestion
def test_load_communities(app, db, location):
    dir_ = Path(__file__).parent
    service = current_communities.service
    communities = CommunitiesFixture(
        [dir_ / "app_data", dir_.parent.parent / "invenio_rdm_records/fixtures/data"],
        "communities.yaml",
        dir_ / "app_data" / "img",
    )
```

minor: instead of defining a `Location` here, you can re-use the `location` fixture (coming from [`pytest-invenio`](https://github.com/inveniosoftware/pytest-invenio/blob/12aafb80acc61fe54e1575ecc125e0c854fdb100/pytest_invenio/fixtures.py#L643)).